### DISCUSSION TITLE
data valutazione

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#649] [Idee e proposte]
Buongiorno ,sarebbe possibile far comparire la data della valutazione in questo modo per vedere sin da subito quando si è effettuata una interrogazione o verifica senza cliccare per forza sopra?Grazie mille 

Allego l'idea 

<img width="1325" alt="Screenshot 2024-02-13 alle 11 33 03" src="https://github.com/iisgiua/giuaschool/assets/93823125/fea41062-375c-4060-abd7-b5af23f5d6a3">

**Comment by trinko** [comment]
La segno come richiesta.
